src/application/scene/fxml/Todo.fxml

```diff
@@ -1,11 +1,91 @@
-   </children>
+<AnchorPane prefHeight="810.0" prefWidth="1315.0" style="-fx-background-color: grey;" stylesheets="@../css/Todo.css" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1" fx:controller="application.scene.Todo">
+    <children>
+        <BorderPane prefHeight="810.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0">
+            <top>
+                <HBox prefHeight="60.0" styleClass="navigation_bar" BorderPane.alignment="CENTER">
+                    <children>
+                        <Button fx:id="buttonDashboard" graphicTextGap="20.0" minHeight="-Infinity" minWidth="-Infinity" mnemonicParsing="false" onAction="#dashboard" prefHeight="60.0" prefWidth="200.0" styleClass="button_tool" text="dashboard">
+                            <graphic>
+                                <ImageView fitHeight="30.0" fitWidth="30.0" pickOnBounds="true" preserveRatio="true">
+                                    <image>
+                                        <Image url="@../assets/icon_dashboard_white.png" />
+                                    </image>
+                                </ImageView>
+                            </graphic>
+                        </Button>
+                    </children>
+                </HBox>
+            </top>
+            <center>
+                <AnchorPane maxHeight="-Infinity" maxWidth="1315.0" prefHeight="810.0" prefWidth="1315.0" style="-fx-background-color: white;" BorderPane.alignment="CENTER">
+                    <children>
+                        <AnchorPane layoutX="50.0" layoutY="137.0" prefHeight="493.0" prefWidth="1215.0" style="-fx-background-color: #332E43;" AnchorPane.bottomAnchor="40.0" AnchorPane.leftAnchor="50.0" AnchorPane.rightAnchor="50.0" AnchorPane.topAnchor="137.0">
+                            <children>
+                                <ListView fx:id="listView_TODO" layoutX="58.0" layoutY="66.0" prefHeight="438.0" prefWidth="317.0" style="" styleClass="listView_TODO" AnchorPane.bottomAnchor="25.0" AnchorPane.leftAnchor="55.0" AnchorPane.rightAnchor="845.0" AnchorPane.topAnchor="100.0" />
+                                <ListView fx:id="listView_DOING" layoutX="736.0" layoutY="100.0" prefHeight="438.0" prefWidth="317.0" styleClass="listView_DOING" AnchorPane.bottomAnchor="25.0" AnchorPane.leftAnchor="450.0" AnchorPane.rightAnchor="450.0" AnchorPane.topAnchor="100.0" />
+                                <ListView fx:id="listView_DONE" layoutX="858.0" layoutY="48.0" prefHeight="438.0" prefWidth="317.0" styleClass="listView_DONE" AnchorPane.bottomAnchor="25.0" AnchorPane.leftAnchor="845.0" AnchorPane.rightAnchor="55.0" AnchorPane.topAnchor="100.0" />
+                                <Text fill="WHITE" layoutX="164.0" layoutY="90.0" strokeType="OUTSIDE" strokeWidth="0.0" styleClass="text_listTitle" text="TODO" wrappingWidth="96.75628662109375" />
```

Comment:
Could these heading texts be changed to lowercase to better fit the rest of the app

Reply:
Fixed